doc/config.md

```diff
@@ -226,6 +226,11 @@ Settings list:
   specified, the fallen tests will not be relaunched (by default it's 0).
 
   Note that the same test never be performed in the same browser session.
+  
+* `shouldRetry` — function which defines whether retry needed or not. Should returns `Boolean` value. First argument is `data` which is result of test run. `data` will contains fields:
```

Comment:
`...whether to make a retry. Should return...`
`which is the result of the test run, it's an Object with following fields:`